upgrade/steps-helpers_test.go

```diff
@@ -0,0 +1,48 @@
+package upgrade
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetRepoExpectedLocation(t *testing.T) {
+	ctx := Context{
+		GoPath: "Users/myuser/go",
+	}
+
+	cwd, err := os.Getwd()
+	assert.Nil(t, err)
```

Comment:
Is there a way we can do this so that our tests don't depend on the current working directory? We could make `getRepoExpectedLocation` a pure function: `func(goPath, cwd, repoPath string) string`.